packages/atomic/src/utils/initialization-utils.tsx

```diff
@@ -69,7 +69,8 @@ export function Initialization() {
       }
 
       if (!this[contextProperty]) {
-        return;
+        // TODO: add optional renderLoad() method to render placeholders
+        return `${getElement(this).nodeName.toLowerCase()}_loading`;
```

Comment:
Hmm, so if a component's context is not defined, should we display the node name? or is this temporary and will be replaced with e.g. an animation?

Reply:
Temporary 😄